### ISSUE TITLE
[TTT] Pauta Reunião

### ISSUE THREAD

**Issue by quasemago** [quasemago/zrageservers#145]
Reformulação moderação:
- Abrir outra aplicação para helpers.

Reports:
- Adicionar admins online no report.
- Pensar num sistema de demo ao utilizar o report.
- Notificar o chat dos admins que algum jogador fez um report.

Revisar issue #105 [Special Round].

**Comment by Danielurcino** [comment]
eu acho justo uma outra aplicaçao de helpers pq no server TTT esta tendo poucos admins onlines todos os dias ent sugiro q façam outra aplicaçao de helpes e se tiver uma eu indico Nescauzinn esta online todos os dias no ttt e tem um bom papel como ''admin''

**Comment by danielhidalgojunior** [comment]
Como eu havia dito, um menu de reports in game para agilizar o processo. Ex.:

sm_reports
1. Acusado: Haru, Vítima: Fanzack, Motivo: rdm pls punish
2. Acusado: Haru, Vítima: Ronning, Motivo: me matou do nada

ao selecionar um item, dar as opções de deferir ou indeferir. Caso o admin escolha punir, abrir um menu com rounds predefinidos (1, 2, 4, 6, 10, etc) e por fim a lista de motivos com a opção de escolher escrever no chat como é feito para mandar um report, por exemplo.

Se conseguir integrar isso com o discord de alguma maneira, mostrando na mensagem do report se o cara foi punido, perdoado, admin q aplicou, rounds etc, seria melhor ainda.

**Comment by Tardigrada-TTT** [comment]
Concordo com tudo, se for possível irá agilizar todo o processo de punish.

**Comment by lhsatsuma** [comment]
Temos previsão para quando esta issue será realizada?

**Comment by hcjordao** [comment]
@danielhidalgojunior tu pode criar a sua sugestão em outro issue. Acho super valida :)
